server/utils/referralUtils.ts

```diff
@@ -96,6 +107,32 @@ export default class ReferralUtils {
     return referral.hasReviewedProgrammeHistory && referral.oasysConfirmed && !!referral.additionalInformation
   }
 
+  static statusTagHtml(status: ReferralStatus): string {
+    let colour: TagColour
+    let text: string
+
+    switch (status) {
+      case 'assessment_started':
+        colour = 'yellow'
+        text = 'Assessment started'
+        break
+      case 'awaiting_assesment':
+        colour = 'orange'
+        text = 'Awaiting assessment'
+        break
+      case 'referral_submitted':
+        colour = 'red'
+        text = 'Referral submitted'
+        break
+      default:
+        colour = 'grey'
+        text = status
+        break
+    }
+
+    return `<strong class="govuk-tag govuk-tag--${colour}">${text}</strong>`
```

Comment:
I'm guessing this differs from the task list status tag method - in that it's generating a HTML string directly - because it's used in a table rows method and can't easily be plugged into the Nunjucks tag macro in the view? Might be good to add something to that effect in the commit message to explain the different approach for future reference?